src/app/user-event/assessment-event.component.ts

```diff
@@ -369,4 +373,17 @@ export class AssessmentEventComponent extends ListComponentDirective<AssessmentM
 
     return observable;
   }
+
+  // Get anonymity from saved answers in case of inline style. Anonymity will be refactored in #92.
+  private _checkInlineAnonymity() {
+    for (let i = 0; i < this.list.length; i++) {
```

Comment:
Does we really need to iterate over `index` and by getter property? For me preferable to use `for-of` with `this._list`, and using iteating by `index` in situation that cannot be possible with `for-of` expression.